README.md

```diff
@@ -10,9 +10,14 @@ The CockroachDB Kubernetes Operator deploys CockroachDB on a Kubernetes cluster.
 ## Limitations
 
 - This project is in beta and is not production-ready.
-- The Operator currently runs on GKE.
+- The Operator currently runs on GKE. VMware Tanzu, EKS, and AKS have not been tested.
 - Currently the Kubernetes CA is required to deploy a secure cluster.
 - Expansion of persistent volumes is not yet functional.
+- The Operator has not been tested with [Istio](https://istio.io/).  
+- The Operator does not yet support [multiple Kubernetes clusters for multi-region deployments](https://www.cockroachlabs.com/docs/stable/orchestrate-cockroachdb-with-kubernetes-multi-cluster.html#eks).
+- The Operator does not yet [support Pod Security Policies](https://github.com/cockroachdb/cockroach-operator/issues/216).
```

Comment:
Again.  We support this but have not implemented the automatic creation of it.

Reply:
Fixed.